src/content/reference/react/memo.md

```diff
@@ -4,7 +4,7 @@ title: memo
 
 <Intro>
 
-`memo` lets you skip re-rendering a component when its props are unchanged.
+`memo` memungkinkan Anda melewatkan render ulang komponen apabila props tidak berubah.
```

Comment:
```suggestion
`memo` memungkinkan Anda untuk melewati proses render ulang komponen apabila props tidak berubah.
```

Reply:
@mazipan In the later suggestion, it is `melewati render` without `proses`? Which one to use?

Reply:
Without `process` should be fine.